playground/src/common/defaultConfig.ts

```diff
@@ -7,5 +7,8 @@ export const defaultConfig: string = `{
     "messages": true,
     "schemas": true,
     "errors": true
+  },
+  "expanded":{
+    "messageExamples": false,
   }
```

Comment:
Here you should have valid JSON:

```suggestion
  "expand": {
    "messageExamples": false
  }
```

and probably by this config for playground didn't work for you.